src/main.js

```diff
@@ -1 +1,9 @@
 console.log('Hello World');
+
+function printNumber(number) {
+    for (let i = 1; i <= number; i++) {
+        console.log(i);
+    }
+}
+
+printNumber(10);
```

Comment:
Try doing the same thing with different possible loop types.

Something like this -
```js
function printNumbersWithForLoop() {
  // This you have already implemented
}

function printNumbersWithWhileLoop() {
  // TODO
}

function printNumbersWithForEach() {
  // TODO
}

printNumbersWithForLoop();
printNumbersWithWhileLoop();
printNumbersWithForEach();
```

Reply:
Done